rabbitmq/rabbitmq.go

```diff
@@ -0,0 +1,39 @@
+package rabbitmq
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+type RabbitMQClient struct {
+}
+
+type RabbitMQResponse struct {
+	Messages float64 `json:"messages"`
+}
+
+func (r *RabbitMQClient) GetNumOfMessages(url, credentials string) (int, error) {
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", url, nil)
```

Comment:
Is the URL the same as the amqp endpoint? (the one that connects to port 5672)

Reply:
No is not the same, this is the URL of the RabbitMQ console endpoint (port 15672)

Reply:
Oh, cool. Do you know which permissions are needed for that?

Reply:
A basic user with read permission on queue can obtain the information provided by api.